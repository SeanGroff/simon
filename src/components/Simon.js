// @flow
import React from 'react';
import styled from 'styled-components';
import InnerWrapper from './InnerWrapper';
import CenterConsole from './CenterConsole';

const Wrapper = styled.div`
  display: flex;
  justify-content: center;
  align-items: center;
  background-color: #333333;
  border-radius: 100%;
  box-shadow: 0 0 12px #222222;
  width: 472px;
  height: 472px;
  position: relative;
`;

export default (props: {}) =>
  <Wrapper>
    <InnerWrapper />
    <CenterConsole />
  </Wrapper>;
