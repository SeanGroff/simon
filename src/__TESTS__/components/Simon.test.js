import React from 'react';
import renderer from 'react-test-renderer';
import Simon from '../../components/Simon';

it('renders correctly', () => {
  const component = renderer.create(<Simon />).toJSON();
  expect(component).toMatchSnapshot();
});
