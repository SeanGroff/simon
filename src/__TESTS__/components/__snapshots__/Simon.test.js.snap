// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-htpNat glJWZx"
>
  <div
    className="sc-bdVaJa crGigL"
  />
  <div
    className="sc-bwzfXH gybXmb"
  />
</div>
`;
